packages/@expo/cli/src/export/persistMetroAssets.ts

```diff
@@ -106,6 +109,21 @@ export async function persistMetroAssetsAsync(
   }
 }
 
+export async function createKeepFile(assets: AssetData[], outputDirectory: string): Promise<void> {
```

Comment:
```suggestion
export async function createKeepFileAsync(assets: AssetData[], outputDirectory: string): Promise<void> {
```

we used to have `Async` suffix for async functions